static/app/components/webAuthn/index.tsx

```diff
@@ -0,0 +1,23 @@
+import type {Authenticator} from 'sentry/types/auth';
+
+import {WebAuthnInterface, type WebAuthnInterfaceProps} from './webAuthnInterface';
+
+interface Props {
+  authenticators: Authenticator[];
+  mode: WebAuthnInterfaceProps['mode'];
+  onWebAuthn: WebAuthnInterfaceProps['onWebAuthn'];
+}
+
+export function WebAuthn({authenticators, ...props}: Props) {
+  if (!authenticators.length) {
+    return null;
+  }
+
+  // XXX(epurkhiser): On the backend we are still calling this interface 'u2f',
+  // even tohugh it's actually been updated to completely support webAuthn.
+  return authenticators
+    .filter(auth => auth.id === 'u2f')
+    .map(auth => (
+      <WebAuthnInterface key={auth.id} {...props} challengeData={auth.challenge} />
+    ));
```

Comment:
This was the behavior before when you have multiple webauthn authenticators, just render a few of these components?